librz/analysis/op.c

```diff
@@ -271,107 +271,60 @@ static struct optype {
 	{ RZ_ANALYSIS_OP_TYPE_SWITCH, "switch" },
 	{ RZ_ANALYSIS_OP_TYPE_TRAP, "trap" },
 	{ RZ_ANALYSIS_OP_TYPE_UCALL, "ucall" },
-	{ RZ_ANALYSIS_OP_TYPE_RCALL, "rcall" }, // needs to be changed
-	{ RZ_ANALYSIS_OP_TYPE_ICALL, "ucall" }, // needs to be changed
-	{ RZ_ANALYSIS_OP_TYPE_IRCALL, "ucall" }, // needs to be changed
+	{ RZ_ANALYSIS_OP_TYPE_RCALL, "rcall" }, 
+	{ RZ_ANALYSIS_OP_TYPE_ICALL, "icall" }, 
+	{ RZ_ANALYSIS_OP_TYPE_IRCALL, "ircall" }, 
 	{ RZ_ANALYSIS_OP_TYPE_UCCALL, "uccall" },
 	{ RZ_ANALYSIS_OP_TYPE_UCJMP, "ucjmp" },
 	{ RZ_ANALYSIS_OP_TYPE_UJMP, "ujmp" },
-	{ RZ_ANALYSIS_OP_TYPE_RJMP, "rjmp" }, // needs to be changed
-	{ RZ_ANALYSIS_OP_TYPE_IJMP, "ujmp" }, // needs to be changed
-	{ RZ_ANALYSIS_OP_TYPE_IRJMP, "ujmp" }, // needs to be changed
+	{ RZ_ANALYSIS_OP_TYPE_RJMP, "rjmp" }, 
+	{ RZ_ANALYSIS_OP_TYPE_IJMP, "ijmp" }, 
+	{ RZ_ANALYSIS_OP_TYPE_IRJMP, "irjmp" }, 
 	{ RZ_ANALYSIS_OP_TYPE_UNK, "unk" },
 	{ RZ_ANALYSIS_OP_TYPE_UPUSH, "upush" },
 	{ RZ_ANALYSIS_OP_TYPE_RPUSH, "rpush" },
 	{ RZ_ANALYSIS_OP_TYPE_XCHG, "xchg" },
 	{ RZ_ANALYSIS_OP_TYPE_XOR, "xor" },
 	{ RZ_ANALYSIS_OP_TYPE_CASE, "case" },
 	{ RZ_ANALYSIS_OP_TYPE_CPL, "cpl" },
-	{ RZ_ANALYSIS_OP_TYPE_CRYPTO, "crypto" },
-	{ 0, NULL }
+	{ RZ_ANALYSIS_OP_TYPE_CRYPTO, "crypto" }
 };
 
-RZ_API int rz_analysis_optype_from_string(const char *type) {
+/**
+ * Return the op type corresponding the given name
+ * \param  name       string, name of the optype
+ * \return type       int, id of the op type (one of \link _RzAnalysisOpType \endlink)
+ */
+RZ_API int rz_analysis_optype_from_string(const char *name) {
```

Comment:
1. I recommend to add `RZ_NONNULL` for `name`
2. Add an assert `rz_return_val_if_fail(name, -1);`